x-pack/legacy/plugins/code/model/usage_telemetry_metrics.ts

```diff
@@ -8,4 +8,12 @@ export enum CodeUsageMetrics {
   ENABLED = 'enabled',
   REPOSITORIES = 'repositories',
   LANGUAGE_SERVERS = 'langserver',
+  ADMIN_PAGE_LOAD_COUNT = 'adminPageLoadCount',
```

Comment:
one last comment before merging this PR. Let's separate the metrics for backend and frontend. Moves these new ones to sth like `CodeUsageUiMetrics`?